swap_meet/item.py

```diff
@@ -1,2 +1,35 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=None): 
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            if type(id) is not int:
+                raise TypeError("Type id is int")
+            else:
+                self.id = id
+        if condition:
+            self.condition = float(condition)
+        else:
+            self.condition = 0.0   
+        
+    
+    def get_category(self):
+        return f"{self.__class__.__name__}"
+        
+    def __str__(self):
+
+        return f"An object of type Item with id {self.id}"
+        
+    def condition_description(self):
```

Comment:
Well done-- though now, you probably can see the advantages of a `dict` for this. O(1) lookup, vs, O(n) for the worse case of falling through n conditions in an `if/else` :)